dts/arm/nxp/nxp_rt10xx.dtsi

```diff
@@ -550,6 +550,7 @@
 				interrupts = <102 0>;
 				#pwm-cells = <2>;
 				clocks = <&ccm IMX_CCM_PWM_CLK 0 0>;
+				prescaler = <7>;
```

Comment:
Similar to ST, we should use vendor prefix here, `nxp,prescaler`. @galak we should probably settle some rules on vendor-specific properties (we have many inconsistencies in-tree).

Reply:
I have added the appropriate prefix.